application/templates/static_site/measure.html

```diff
@@ -747,6 +747,11 @@ <h2 id="download-the-data" class="govuk-accordion__section-heading">
         chart_div.classList.remove('hidden');
     };
 
+    var download_pngs = document.getElementsByClassName('download-png');
```

Comment:
We probably need to do some sort of feature-detection here to make sure that the charts are downloadable (ie whatever Highcharts needs?). Otherwise it’d still appear in older browsers (so long as they support `getElementsByClassName` and `classList`)?

Reply:
I did spend a bit of time trying to do this, but it felt painful. Maybe we can pair on it?